gradle.properties

```diff
@@ -10,7 +10,7 @@ org.gradle.jvmargs=-Xmx1G \
 # Fabric Properties
 	# check these on https://fabricmc.net/use
 	minecraft_version=1.17.1
-	yarn_mappings=1.17.1+build.1
+    yarn_mappings=1.17.1+build.1
```

Comment:
Why

Reply:
adverb
for what reason or purpose.
"why did he do it?"
adverb
(with reference to a reason) on account of which; for which.
"the reason why flu jabs need repeating every year is that the virus changes"
exclamation
1.
expressing surprise or indignation.
"why, that's absurd!"
2.
used to add emphasis to a response.
"‘You think so?’ ‘Why, yes.’"
noun
a reason or explanation.
"the whys and wherefores of these procedures need to be explained to students"